### PR TITLE
Danielle oneal

### PR DESCRIPTION
**Pull request by dani-digital**

### CODE REVIEW

File: generic.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	Stellar by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+	<head>
+		<title>Generic - Stellar by HTML5 UP</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
```
Comment (joepound) [inline generic.html]: Don't forget to change up the `<title>` contents to display the title text (displayed most prominently on your browser tab, among other places) that you want.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	Stellar by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+	<head>
+		<title>Stellar by HTML5 UP</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
```
Comment (joepounddev) [inline index.html:12]: Don't forget to change up the `<title>` contents to display the title text (displayed most prominently on your browser tab, among other places) that you want.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+								<dd>273 Overland Way &bull; Gray, GA 31032  &bull; USA</dd>
+								<dt>Phone</dt>
+								<dd>(478) 607-9358</dd>
+								<dt>Email</dt>
+								<dd><a href="#">contact@danionealdev.com</a></dd>
+							</dl>
+							<ul class="icons">
+								<li><a href="facebook.com/digitalyelldesigns" class="icon fa-facebook alt"><span class="label">Facebook</span></a></li>
+								<li><a href="https://www.instagram.com/digitalyelldesigns/" class="icon fa-instagram alt"><span class="label">Instagram</span></a></li>
+								<li><a href="https://github.com/droneal11015" class="icon fa-github alt"><span class="label">GitHub</span></a></li>
```
Comment (joepounddev) [inline index.html:127]: Heads-up for your Facebook link: you'll also want to add "https://www." at the start of the `href` attribute value to make sure that the link will always redirect to the actual Facebook page that you want.

**Review by joepounddev — COMMENTED** [review]
Great work in setting up the portfolio page! Content integrated well into the template and it is also responsive too!

Nice touch with adding the appropriate button/icon links too! In the future, if you so choose, you can also display screenshots (or even live demos) of some projects of yours that you're particularly proud of and want to feature on your page (aside from the GitHub link that lists all your projects).

Lastly, it helps to commit more often (with matching descriptive commit messages too) - perhaps each time you finish a certain portion of your page(s).

The page looks great overall and it would be cool for you to deploy it in GitHub Pages!

**Pull request merged**